app/tests/v1/redflags_test.py

```diff
@@ -0,0 +1,74 @@
+from unittest import TestCase
```

Comment:
What is the difference between this file and [this](app/tests/redflags_test.py). Put all redflags tests in one module